src/patternfly/components/Chip/chip.scss

```diff
@@ -79,22 +82,28 @@
     }
   }
 
-  // Button
-  .pf-c-button {
-    --pf-c-button--PaddingTop: var(--pf-c-chip__c-button--PaddingTop);
-    --pf-c-button--PaddingRight: var(--pf-c-chip__c-button--PaddingRight);
-    --pf-c-button--PaddingBottom: var(--pf-c-chip__c-button--PaddingBottom);
-    --pf-c-button--PaddingLeft: var(--pf-c-chip__c-button--PaddingLeft);
-    --pf-c-button--FontSize: var(--pf-c-chip__c-button--FontSize);
-
-    margin-top: var(--pf-c-chip__c-button--MarginTop);
-    margin-right: var(--pf-c-chip__c-button--MarginRight);
-    margin-bottom: var(--pf-c-chip__c-button--MarginBottom);
+  .pf-c-chip__content {
+    display: flex;
+    column-gap: var(--pf-c-chip__content--ColumnGap);
+    font-size: var(--pf-c-chip__content--FontSize);
+  }
+
+  .pf-c-chip__actions {
+    font-size: var(--pf-c-chip__actions--FontSize);
   }
 
-  // badge
-  .pf-c-badge {
-    margin-left: var(--pf-c-chip__c-badge--MarginLeft);
+  // Actions Button
+  .pf-c-chip__actions .pf-c-button {
```

Comment:
Can you nest this as `.pf-c-button { ... }` under the existing `.pf-c-chip__actions` selector?